packages/job-manager/tsconfig.json

```diff
@@ -1,6 +1,10 @@
 {
     "extends": "../../tsconfig.base.json",
+    "lib": ["es6", "dom"],
+    "types": ["reflect-metadata"],
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
     "compilerOptions": {
-        "outDir": "out"
+        "outDir": "dist"
     }
```

Comment:
will this override all compilerOptions set on tsconfig.base? (I am assuming it won't)
why not move this to base, so that, you don't have to repeat this everywhere?

Reply:
moving to base breaks repo build. need more investigation.